### ISSUE TITLE
sbt 1.0.x Support

### ISSUE THREAD

**Issue by danielnixon** [unicredit/sbt-swagger-codegen#56]
Please support sbt 1.0.x.

**Comment by mfirry** [comment]
I'm not so sure how one can keep publishing a plugin for both 0.13.x and 1.x... @andreaTP any thoughts?

**Comment by mfirry** [comment]
- [ ] upgrade `sbt-sonatype`
- [ ] upgrade `sbt-pgp`
- [ ] upgrade `sbt-scalafmt`

**Comment by andreaTP** [comment]
sure a couple of sbt incantations will help, e.g.
```
sbt ;set sbtVersion := "0.13.16";set scalaVersion := "2.10.6";publish
sbt ;set sbtVersion := "1.0.4";set scalaVersion := "2.12.4";publish
```

in `build.sbt` you can use something like:
```scala
scalaCrossVersions := Seq("2.10.6", "2.12.4")
sbtCrossVersions := Seq("0.13.16", "1.0.4")
```

**Comment by mfirry** [comment]
Ok.

I've upgraded all the plugins and now I get a compilation error here: https://github.com/unicredit/sbt-swagger-codegen/blob/master/plugin/src/main/scala/eu/unicredit/swagger/SwaggerCodegenPlugin.scala#L124

What's this `.***.` magic?

**Comment by mfirry** [comment]
Fine, I've pushed the little changes required for sbt 1.0.x. Can someone have a look? @andreaTP? @fralken?

**Comment by fralken** [comment]
fixed by #59

**Comment by dvirf1** [comment]
updated [sbt 1.x plugin migration](https://github.com/sbt/sbt/wiki/sbt-1.x-plugin-migration) page
